dindex_builder/graph_index_schema_kuzu.txt

```diff
@@ -1,3 +1,3 @@
-CREATE NODE TABLE Column(id INT64, PRIMARY KEY (id));
+CREATE NODE TABLE Dataset(id INT64, PRIMARY KEY (id));
```

Comment:
I think using the name "Dataset" is a bit misleading, maybe we can use "DataColumn" or "DatasetColumn" or something else that can show that node represents column.

Reply:
I like ColumnNode, it indicates a type of node in the graph, e.g., column.